controls/radwatermarktextbox/getting-started.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Getting Started
+page_title: Getting Started
+description: Getting Started
+slug: watermarktextbox-getting-started
+tags: getting,started
+published: True
+position: 1
+---
+
+# Getting Started
+
+This tutorial will walk you through the required steps for using __RadWatermarkTextBox__. 
+
+* [Assembly References](#assembly-references)
+
+## Assembly References
+
+In order to use __RadWatermarkTextBox__ in your application, you need to add references to the following assembly:
+
+* __Telerik.Windows.Controls__

```

Comment:
You can move this in the sentence above. 

In order to use __RadWatermarkTextBox__ in your application, you need to add references to the __Telerik.Windows.Controls__ assembly.